crates/fluvio/Cargo.toml

```diff
@@ -40,7 +40,11 @@ siphasher = "0.3.5"
 cfg-if = "1.0.0"
 derive_builder = "0.10"
 async-trait = "0.1.51"
-
+sysinfo = "0.24"
```

Comment:
In the previous conversation, I requested that these additional dependencies should be under the feature flag and ensure they work in the web assembly.